### ISSUE TITLE
Currency codelist has incorrect external link

### ISSUE THREAD

**Issue by ekoner** [open-contracting/standard#297]
The [Currency codelist](http://standard.open-contracting.org/latest/en/schema/codelists/) is described as: 

"OCDS uses the ISO 3-letter currency codes maintained in ISO 4217"

The link points to wikipedia, shouldn't it point to http://www.iso.org/iso/home/standards/currency_codes.htm?

Could we also make the description more contextual, a la:

"Release Tag

A contracting process may result in a number of releases of information over time. These should be tagged to indicate the stage of the contracting process they relate to."

**Comment by timgdavies** [comment]
I believe when we did this the ISO link provided no access to a machine-readable version of the codes.

I see that they now provide this at http://www.currency-iso.org/dam/downloads/lists/list_one.xml which is linked from http://www.iso.org/iso/home/standards/currency_codes.htm

I propose that we update the text to:

"The currency for each amount should always be specified using the uppercase 3-letter currency code from [ISO4217](http://www.iso.org/iso/home/standards/currency_codes.htm)." 

Updated approaches to codelist management may have an impact on whether the codelist remains external or is incorporated into the standard. Assigning to @kindly to consider this within codelist proposals.
